gef.py

```diff
@@ -2593,6 +2593,79 @@ def inner_f(*args, **kwargs):
     return wrapper
 
 
+def FakeExit(*args, **kwargs):
+    raise RuntimeWarning
+
+sys.exit = FakeExit
+
+def parse_arguments(required_arguments, optional_arguments):
+    """Argument parsing decorator."""
+
+    def int_wrapper(x): return int(x, 0)
+
+    def decorator(f):
+        def wrapper(*args, **kwargs):
```

Comment:
Can we use `functools.wraps`? This does some magic that makes it so that stack traces don't show the decorator.

Otherwise our stack traces are harder to read. Example:

```python
↳ File "~/code/gef/gef.py", line 6335, in do_invoke()
    →             raise Exception
↳ File "~/code/gef/gef.py", line 2646, in wrapper()
    →                 return f(*args, **kwargs)
↳ File "~/code/gef/gef.py", line 247, in wrapper()
    →                 rv = f(*args, **kwargs)
↳ File "~/code/gef/gef.py", line 4192, in invoke()
    →                 bufferize(self.do_invoke)(argv)
```

Reply:
Good call, I forgot about `functools.wraps`.